packages/editor/src/components/block-vertical-alignment-toolbar/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { Toolbar } from '@wordpress/components';
+import { withViewportMatch } from '@wordpress/viewport';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { valignTop, valignCenter, valignBottom } from './icons';
+
+/**
+ * Internal dependencies
+ */
+import { withBlockEditContext } from '../block-edit/context';
+
+const BLOCK_ALIGNMENTS_CONTROLS = {
+	top: {
+		_id: 'top',
+		icon: valignTop,
+		title: _x( 'Vertically Align Top', 'Block vertical alignment setting' ),
+	},
+	center: {
+		_id: 'center',
+		icon: valignCenter,
+		title: _x( 'Verticaly Align Middle', 'Block vertical alignment setting' ),
```

Comment:
Minor: The copy for center and bottom seems to be missing an `l`: `Verticaly` => `Vertically`